content/docs/dvclive/api-reference/live/index.md

```diff
@@ -71,6 +74,9 @@ You can use `Live()` as a context manager. When exiting the context manager,
 
   If `report` is `None`, `Live.make_report()` won't generate anything.
 
+- `save_dvc_exp` - If `True`, DVCLive will create a new <abbr>DVC
+  experiment</abbr> as part of `Live.end()`. Defaults to `False`.
```

Comment:
Since there's no method to link to, maybe we should describe the behavior here and link back to here from live.next_step, live.end, and wherever else we mention this behavior.

Reply:
Agree on having a single section to reference but I was thinking that it could make more sense maybe in `How it Works`, wdyt?

Reply:
Let me take another look 🤔 😓 . 

Forgot to mention that we also should specify somewhere that it turns off if there is an existing dvc.yaml.

Reply:
Yup, ended up adding it to `How it Works`.